### ISSUE TITLE
Serene Seasons Incompatible

### ISSUE THREAD

**Issue by Mikeytcman** [iPortalTeam/ImmersivePortalsModForForge#76]
### According to your testing, does the issue occur with ONLY Immersive Portals mod installed?

No

### Forge Version

1.19.2-Forge-43.2.3

### ImmersivePortals Version

immersive-portals-2.3.1-mc1.19.2-forge

### Latest Log

https://gist.githubusercontent.com/Mikeytcman/3aee2e9afc243e5cf511882b8f5d3fc0/raw/649a0aee435f0d51b8a40edbdad8ee2a4d16155e/debug.log

### Crash Report (if applicable)

https://gist.githubusercontent.com/Mikeytcman/3aee2e9afc243e5cf511882b8f5d3fc0/raw/649a0aee435f0d51b8a40edbdad8ee2a4d16155e/crash-2023-01-24_10.37.41-server.txt

### Steps to Reproduce

1. Add [immersive-portals-2.3.1-mc1.19.2-forge.jar](https://www.curseforge.com/minecraft/mc-mods/immersive-portals-for-forge/files/4352251) to mods folder
2. Add [Serene Seasons 1.19.2-8.1.0.24](https://www.curseforge.com/minecraft/mc-mods/serene-seasons/files/4037228) to mods folder
3. launch minecraft with 1.19.2-Forge-43.2.3
4. create a new world or load existing world

### What You Expected

The world should load world without issue as normal

### What Happened Instead

The game instantly crashed when pressing Done on create screen or play on existing world

### Additional Details

I have to click "I have confirmed this does not happen on the fabric version (If it does please report to [here](https://github.com/iPortalTeam/ImmersivePortalsMod/issues) instead)." Even though I don't use fabric so IDK if this is an issue on Fabric or not.

### Please Read and Confirm The Following

- [X] I have confirmed this bug can be replicated without the use of Optifine.
- [X] I have confirmed the details provided in this report are concise as possible and does not contained vague information (ie. Versions are properly recorded, answers to questions are clear).
- [x] I have confirmed this does not happen on the fabric version (If it does please report to **[here](https://github.com/iPortalTeam/ImmersivePortalsMod/issues)** instead).
- [X] I have confirmed I'm not using a hybrid server (Magma, Cauldron) or a different (non official) Forge Server release.
- [X] I have confirmed this issue is unique and has not been reported already.

**Comment by Lolothepro** [comment]
#14

**Comment by ghost** [comment]
They said in Discord that you can get access for it

**Comment by Lolothepro** [comment]
![image](https://user-images.githubusercontent.com/68275908/215277853-f48441e1-0ed0-471d-ba1e-3e72bf8a2cda.png)

**Comment by Nick1st** [comment]
Yeah, I have better things to do than asking them for their source code.

**Comment by MarioSMB** [comment]
Whatever their reasoning, the decision to go closed-source only hurts the community - see the impact OptiFine being closed-source has had over the years and the reputation it has garnered.  
In that regard I don't blame you for not wanting to cater to their strange requirement, but if it would help I don't mind reaching out to the Serene Seasons developers on your behalf to request access to their source code.

**Comment by Nick1st** [comment]
This issue is for an outdated Minecraft version that is no longer supported. 

This issue has been added to the list [1.19.2 - Known Issues](https://github.com/iPortalTeam/ImmersivePortalsModForForge/discussions/220).

This issue will be marked with an `Unsolved` label to mark it as an active qualified issue besides being closed as `Not planned`.

If this issue persists in a newer Minecraft version, please open a new issue.
